### DISCUSSION TITLE
achievement

### DISCUSSION THREAD

**Discussion by Arisaputraa19** [badges/shields#10751] [Q&A] [answered]
how to get galaxy brain achievement on github?

**Comment by Doriss007** [comment, marked as answer]
to achieve the 'Galaxy Brain' achievement, take note of the following:

1. Answer at least 2 discussions.
2. Take note, you CANNOT mark your own answers to your own questions. They have to be marked by another user.
3. Discussions should be in a public repository.
4. There's a current bug observed by some users whereby marking your own answers can prevent you from getting the badge instead.

Hope this answer has helped you, and all the best in getting the achievement! :)
Reference: https://github.com/orgs/community/discussions/18467

**Reply by Arisaputraa19** [reply]
anything else besides that?

**Reply by Doriss007** [reply]
That's all I know, maybe other people have something else they can tell you.

**Reply by Arisaputraa19** [reply]
okayy, thanks
